package.json

```
@@ -27,16 +27,21 @@
 		"watch:client": "tsc -w -p client/tsconfig.json",
 		"compile:server": "tsc -p server/tsconfig.json",
 		"watch:server": "tsc -w -p server/tsconfig.json",
-		"clean:server": "rm -fr ./server/node_modules && rm -fr ./server/out",
-		"clean:client": "rm -fr ./client/node_modules && rm -fr ./client/out",
-		"clean": "npm run clean:server && npm run clean:client && rm -fr node_modules",
+		"clean:server": "rm -fr ./server/node_modules && rm -fr ./server/out && rm -rf server/tsconfig.tsbuildinfo && rm -fr server/.vscode-test",
```

do you think writing down `rm -rf` just once could make this much more readable?
```suggestion
		"clean:server": "rm -fr ./server/node_modules ./server/out server/tsconfig.tsbuildinfo server/.vscode-test",
```

---

Done